logger.go

```diff
@@ -9,18 +9,20 @@ import (
 	"github.com/gobuffalo/pop/v6/logging"
 )
 
-type logger func(lvl logging.Level, s string, args ...interface{})
```

Comment:
the internal definition is almost meaningless since there is only one implementation and it is not exposed.